middleware.go

```diff
@@ -274,3 +275,19 @@ func listDir(t *template.Template, name string, res *echo.Response) (err error)
 	}
 	return t.Execute(res, data)
 }
+
+// cors middleware handler adds cors to app paths
+func EnableCORS(method string, path string) echo.MiddlewareFunc {
+	if method == "PUT" {
+		return middleware.CORSWithConfig(middleware.CORSConfig{
+			AllowOrigins: []string{path},
+			AllowHeaders: []string{"*"},
+			AllowMethods: []string{http.MethodPut, http.MethodGet, http.MethodOptions},
```

Comment:
Not sure you're using the allowed method correctly.

Reply:
Specifically put methods need get along with options with regards to cors for the request to be successful other than that i think the other methods can be just by themselves hence the other block just returns with the method.